examples/treasure-hunt/README.md

```diff
@@ -0,0 +1,46 @@
+# Treasure Hunt
+
+## Overview
+
+This Flask application leverages GPTScript to generate clues for a real world treasure hunt game. The user needs to provide a list of locations within a city and the application will generate fun and interesting clues for them.
+
+## Installation
+
+### Prerequisites
+
+- Python 3.8 or later
+- Node.js and npm
+- Flask
+- Other Python and Node.js dependencies listed in `requirements.txt` and `package.json` respectively.
+
+### Steps
+
+1. Clone the repository:
+
+    ``` bash
+    git clone https://github.com/gptscript-ai/gptscript.git
+    ```
+
+2. Navigate to the `examples/treasure-hun` directory and install the dependencies:
```

Comment:
typo, missing the 't' at the end of hunt.

Reply:
Done.